README.md

```diff
@@ -1,5 +1,7 @@
-ODBC FDW for PostgreSQL 9.5+ [![Build Status](https://travis-ci.org/CartoDB/odbc_fdw.svg?branch=master)](https://travis-ci.org/CartoDB/odbc_fdw)
+ODBC FDW for PostgreSQL 9.5+ 
 ============================
+[![Travis Build Status](https://travis-ci.org/CartoDB/odbc_fdw.svg?branch=master)](https://travis-ci.org/CartoDB/odbc_fdw)
+[![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/mlt/odbc_fdw?branch=appveyor&svg=true)](https://ci.appveyor.com/project/PurpleI2P/odbc-fdw "Get your fresh Windows build here!")
```

Comment:
Pls change the URL to the new AppVeyor set up:
https://ci.appveyor.com/api/projects/status/github/CartoDB/odbc_fdw?branch=master&svg=true